demo-forms/custom_contacts/README.MD

```diff
@@ -0,0 +1,11 @@
+# Applying Sample Custom Contact Types
+
+
+1. Download the current `app_settings.json`
+1. Replace the existing `contact_types` with the contents of [contact_types.json](contact_types.json)
+1. Upload the change to `app_settings.json`
+1. Download the english language file.
```

Comment:
I actually don't know what this step is/how to do it.  Link? More info? Maybe [this file](https://github.com/medic/cht-core/blob/master/config/default/translations/messages-en.properties)?

Reply:
You can download the language file from the admin app, modify it, and then upload in the same page.